source/caracteristicas.py

```diff
@@ -1,54 +1,59 @@
 # -*- coding: utf-8 -*-
 
-from funciones import caracteristicaValida
+from source.funciones import caracteristicaValida
 
 class Caracteristicas:
     
 
-    def __init__(self,accion,terror,romance,comedia,efectos_especiales, valoracion):
+    def __init__(self,accion,terror,romance,comedia,efectos_especiales):
         
         #Todas las características serán valores reales del [0.0 - 5.0]
-        self.accion = caracteristicaValida(accion)
-        self.terror = caracteristicaValida(terror)
-        self.romance = caracteristicaValida(romance)
-        self.efectos_especiales = caracteristicaValida(efectos_especiales)
-        self.valoracion = caracteristicaValida(valoracion)
+        caracteristicaValida(accion)    #En caso de no ser válida genera un error
+        caracteristicaValida(terror)
+        caracteristicaValida(romance)
+        caracteristicaValida(comedia)
+        caracteristicaValida(efectos_especiales)
+        
+        self.atributos={}
+        
+        self.atributos["accion"] = accion
```

Comment:
Si pasaras un diccionario en vez de un conjunto de variables:
1. Podrías hacer un bucle
2. Podrías cambiar fácilmente las características y añadir (o borrar)
Por eso es por lo que tenéis que plantear los issues como un problema, y no partir de la preconcepción de lo que creéis que tenéis que hacer.

Reply:
El problema de pasar un diccionario es que el que usase la clase desde fuera debe saber que atributos ponerle al diccionario, de la manera en que esta puesto puedo controlar que sean 5 valores, y cuales.
Pensé hacerlo de esta manera para restringir más como usar la clase, imagínate (pasando un diccionario por parámetro) que se crean distintos objetos caracteristica con tamaño distinto, para que no petase luego la función de estimación por ejemplo, debo controlar que la matriz de datos sea cuadrada. Por eso pensé en hacerlo así.

Otro problema que se me ocurre es que no llame a las claves siempre igual, o que cambiase el orden de las características en algunos objetos y en otros no,etc